### DISCUSSION TITLE
Proxmox LXC Updater Question

### DISCUSSION THREAD

**Discussion by Photo64** [tteck/Proxmox#852] [Q&A] [answered]
I have a few LXC containers (node red, unifi etc) that I haven't updated the underlying OS since I installed them. If I use the LXC Updater script will this just auto update all LXC containers or does it provide a way that asks which containers should be updated as it goes through the script?  If not, is there a way to individually update each container?

**Comment by tteck** [comment, marked as answer]
All or nothing with the script. But an `apt update && apt upgrade -y` in each LXC console wins 😉
